src/snowflake/snowpark/functions.py

```diff
@@ -6757,7 +6763,7 @@ def udtf(
 
     Args:
         handler: A Python class used for creating the UDTF.
-        output_schema: A list of column names, or a :class:`~snowflake.snowpark.types.StructType` instance that represents the table function's columns.
+        output_schema: A list of column names, or a :class:`~snowflake.snowpark.types.StructType` instance that represents the table function's columns, or a `PandasDataFrameType` instance for vectorized UDTF.
```

Comment:
```suggestion
        output_schema: A list of column names, or a :class:`~snowflake.snowpark.types.StructType` instance that represents the table function's columns, or a ``PandasDataFrameType`` instance for vectorized UDTF.
```
can you double check this but I think you need double backticks here. and other descriptions below